CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.30.0](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/compare/3.29.0...3.30.0) - 2021-02-24
+### Added
+- [[#1285](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1285)] Added support for PHP 8.0 ([@Yozhef](https://github.com/Yozhef))
+
+### Deprecated
+- [[#1291](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1291)] Passing another `type` value to a filter than an integer handled ([@VincentLanglet](https://github.com/VincentLanglet))
+
+### Fixed
+- [[#1314](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1314)] Unavoidable deprecation about the `code` option. ([@VincentLanglet](https://github.com/VincentLanglet))
+- [[#1247](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1247)] `ChoiceFilter` now returns `null` values when used with the type `NOT_EQUAL` ([@VincentLanglet](https://github.com/VincentLanglet))
```

Comment:
IMO, the term "now" is misleading in documents intended to register a timeline of historic events.